lib/fields.py

```diff
@@ -19,6 +19,27 @@ def clean(self, value, model_instance):
         return value
 
 
+class SelectManyField(forms.ModelMultipleChoiceField):
```

Comment:
This field class name seems to be from the old version, so you somehow did not update it here.